enos/modules/worker/main.tf

```diff
@@ -203,3 +221,19 @@ resource "enos_boundary_start" "worker_start" {
     }
   }
 }
+
+resource "enos_remote_exec" "update_directory" {
+  depends_on = [enos_boundary_start.worker_start]
```

Comment:
This seems odd to me, wouldn't the worker fail to start if the permissions are not correct? Shouldn't the dependency be the other way around?

Reply:
It seemed to start ok if the permissions were wide open. This update step has a dependency on `worker_start` to ensure that the boundary user is available.